File: PDOO-Napakalaki/Napakalaki/NapakalakiRuby/pkg/Napakalaki-0.0.1/lib/prize.rb
# To change this license header, choose License Headers in Project Properties.
# To change this template file, choose Tools | Templates
# and open the template in the editor.

class Prize
  
  attr_accessor :treasures, :level
  
  def initialize(treasures, level)
    @treasures = treasures
    @level = level    
  end  
  
  def to_s
    "\nTesoros Ganados: #{@treasures} \n Niveles Ganados: #{@level}"
  end
  
end

File: PDOO-Napakalaki/Napakalaki/NapakalakiJava/src/NapakalakiGame/Prize.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package NapakalakiGame;

/**
 *
 * @author ramonjtorres
 */
public class Prize {


    private int treasures; 
    private int level; 
    
    public Prize(int treasures, int level ){
        this.treasures=treasures;
        this.level=level;
        
    }
    
    public int getTreasures(){
        return treasures;     
    }
    
    public int getLevel(){
        return level; 
    }
    @Override
    public String toString() {
        return "Ganas " + treasures+ " tesoros." + "\nGanas " + level + " niveles.\n";
    }

    
    
}
